src/js/components/Tab/Tab.js

```diff
@@ -7,30 +7,48 @@ import { Box } from '../Box';
 import { Button } from '../Button';
 import { Text } from '../Text';
 import { normalizeColor } from '../../utils';
+import { TabsContext } from '../Tabs/TabsContext';
 
 import { StyledTab } from './StyledTab';
 
 const Tab = forwardRef(
   (
-    {
-      active,
-      icon,
-      plain,
-      title,
-      onActivate,
-      onMouseOver,
-      onMouseOut,
-      reverse,
-      ...rest
-    },
+    { children, icon, plain, title, onMouseOver, onMouseOut, reverse, ...rest },
     ref,
   ) => {
+    const {
+      active,
+      activeIndex,
+      setActiveContent,
+      setActiveTitle,
+      onActivate,
```

Comment:
Nit: recommend alphabetical order

```suggestion
      active,
      activeIndex,
      onActivate,
      setActiveContent,
      setActiveTitle,
```